server/opendp_apps/analysis/validate_release_util.py

```diff
@@ -169,7 +172,15 @@ def run_release_process(self):
         for stat_spec in self.stat_spec_list:
 
             # Okay! -> run_chain(...)!
-            file_handle = open(filepath, 'r')
+            file_extension_determiner = FileExtensionDeterminer(filepath)
+            file_extension = file_extension_determiner.get_file_extension()
+            if file_extension not in ['.csv', '.tab']:
+                spss_reader = SpssReader(filepath)
+                df = spss_reader.read()
+                df.to_csv('temp.csv')
```

Comment:
This file is being written within the code structure, under "server/"
In addition, it's not being cleaned up afterward